test/common/testlib.py

```diff
@@ -766,6 +766,16 @@ def is_nondestructive(self):
         test_method = getattr(self.__class__, self._testMethodName)
         return getattr(test_method, "_testlib__non_destructive", False)
 
+    @staticmethod
+    def disable_preload(machine, *packages):
```

Comment:
Let's not make these static; we don't need them outside of instances, and we may want to use the non-destructive variants at some point.

Reply:
indeed, I already had another thought about it and actually use `self.write_file`.